problems/arrayMethods.js

```diff
@@ -91,15 +141,32 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
+const counterObject = (items) => {
+  let newObject = {};
+  items.forEach((el) => {
+    newObject[el] = newObject[el] + 1 || 1;
+    // if (newObject[el]) {
+    //   newObject[el] += 1;
+    // } else {
+    //   newObject[el] = 1;
+    // }
+  });
+  return newObject;
+};
 
 /**
  * Takes in an array of numbers and returns a new array with each element doubled.
  * @param {number[]} nums
  * @returns {number[]} Each element has been doubled
  */
 
-const doubled = () => {};
+const doubled = (nums) => {
```

Comment:
This should be done with a map. It's returning a new array with each element acted on by the callback.